web/src/components/DiffCard/CircularProgressWithLabel.jsx

```diff
@@ -47,11 +51,11 @@ export default function CircularProgressWithLabel(props) {
         variant="determinate"
         thickness={4}
         className={clsx(styles.top, {
-          [styles.badStatus]: value >= 75,
-          [styles.mediumStatus]: value >= 50 && value < 75,
-          [styles.goodStatus]: value < 50
+          [styles.badStatus]: progress >= 75,

```

Comment:
I think you need to add `showColorBasedStatus` for each class name. eg.g. `showColorBasedStatus && progress >= 75`

Reply:
facepalm

Reply:
clsx discards falsy values, it works